### PR TITLE
Change neighbors failed to connect messages from info to debug

### LINKED ISSUE #1912: Remove noisy neighbors
**Is your feature request related to a problem? Please describe.**

While running a node there are a ton of `PeerNotConnected` messages that appear in the logs, and make it hard to follow other operations. 

**Describe the solution you'd like**

I would like to see the messages silenced but still appear if running with debug enabled. I am planning to submit a PR following this issue.

**Describe alternatives you've considered**

It could be left as-is, but if they are not blocking the node's operation the logs would be much cleaner if these were moved over to debug.

**Additional context**

20200925_001_Selection

### PR DESCRIPTION
**Pull request by whoabuddy**
Fixes #1912 

Looks like my editor also removed a lot of whitespace from the file, if this is an issue I can resubmit, but the actual lines changed are: L1742 and L1917.

### CODE REVIEW

**Comment by diwakergupta** [comment]
> Fixes #1912
> 
> Looks like my editor also removed a lot of whitespace from the file, if this is an issue I can resubmit, but the actual lines changed are: L1742 and L1917.

Yeah I think you may want to re-submit. We should clean up dangling whitespace in a separate PR. Thanks!

**Comment by whoabuddy** [comment]
@diwakergupta fixed in a72e996

**Review by jcnelson — APPROVED** [review]
LGTM.  Thanks @whoabuddy!
